### ISSUE TITLE
[dev.icinga.com #7939] Use an author-supplied, highly visible focus indicator

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1226]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7939

**Created by elippmann on 2014-12-03 10:15:28 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-02-19 16:06:04 +00:00_)
Target Version: _Accessibility_
Last Update: _2016-02-19 16:06:04 +00:00 (in Redmine)_

---

http://www.w3.org/TR/WCAG20-TECHS/general.html#G195 wrote:
> The objective of this technique is enhance the focus indicator in the browser, by creating a highly visible one in the content. The default focus indicator in many browsers is a thin,dotted, black line. It can be difficult to see the line when it is around a form element which already has an outline, when the focused element is inside a table cell, when the focused element is very small, or when the background of the page is a dark color.
> 
> In this technique, when the user places focus on an element, using the mouse, tab key, arrow keys, keyboard shortcuts, or any other method, the application makes that focus more visible, using a combination of a highly contrasting color, a thick line, and other visual indicators such as a glow.

**Acceptance Criteria:**

-   Place focus on each focusable user interface element on the page using the mouse.
-   Check that there is a highly visible focus indicator.
-   Place focus on each focusable user interface element on the page using the keyboard.
-   Check that there is a highly visible focus indicator.
